docs/codeql/codeql-overview/codeql-changelog/codeql-cli-2.20.5.rst

```diff
@@ -109,14 +109,6 @@ Python
 *   Fixed a bug in the extractor where a comment inside a subscript could sometimes cause the AST to be missing nodes.
 *   Using the :code:`break` and :code:`continue` keywords outside of a loop, which is a syntax error but is accepted by our parser, would cause the control-flow construction to fail. This is now no longer the case.
 
-Major Analysis Improvements
-~~~~~~~~~~~~~~~~~~~~~~~~~~~
-
-Golang
-""""""
-
-*   Go 1.24 is now supported. This includes the new language feature of generic type aliases.
```

Comment:
We do want this to be in, but I think we can add this again when we re-generate the changelog so we can drop it for now.